tsconfig.json

```
@@ -15,7 +15,7 @@
     "tsBuildInfoFile": "./tsBuildInfo.json",
     "removeComments": false,
     "importHelpers": true,
-    "downlevelIteration": false,
+    "downlevelIteration": true,
```

Should be able to avoid this config change per my previous suggestion.
```suggestion
    "downlevelIteration": false,
```